components/user/UserRepos.js

```diff
@@ -1,56 +1,64 @@
 import FallbackImage from "@components/FallbackImage";
 import Link from "@components/Link";
+import Delete from "@components/account/manage/Delete";
 import ChevronRightIcon from "@heroicons/react/20/solid/ChevronRightIcon";
 import StarIcon from "@heroicons/react/20/solid/StarIcon";
 import dateFormat from "@services/utils/dateFormat";
 
-export default function UserRepos({ repos }) {
+export default function UserRepos({ manage = false, handleDelete, repos }) {
+  const item = (repo) => (
+    <div className="relative flex justify-between gap-x-6 px-4 py-5 hover:bg-gray-50 sm:px-6 lg:px-8">
+      <div className="flex gap-x-4">
+        <FallbackImage
+          className="h-12 w-12 flex-none rounded-full bg-gray-50"
+          src={`https://github.com/${repo.owner}.png`}
+          alt={`${repo.owner} avatar`}
+          width={100}
+          height={100}
+        />
+        <div className="min-w-0 flex-auto">
+          <p className="text-sm font-semibold leading-6 text-gray-900">
+            <Link href={repo.url} target="_blank">
+              <span className="absolute inset-x-0 -top-px bottom-0" />
+              {repo.owner}/{repo.name}
+            </Link>{" "}
+            <span className="hidden md:inline">
+              ({dateFormat({ format: "long", date: repo.dates.pushedAt })})
+            </span>
+          </p>
+          <p className="mt-1 flex text-xs leading-5 text-gray-500">
+            {repo.description}
+          </p>
+        </div>
+      </div>
+      <div className="flex items-center gap-x-4">
+        <div className="hidden sm:flex sm:flex-col sm:items-end">
+          <p className="inline-flex gap-2 p-3 items-center content-center rounded-md bg-yellow-50 px-2 py-1 text-xs font-medium text-yellow-800 ring-1 ring-inset ring-yellow-600/20">
+            <StarIcon
+              className="h-4 w-4 flex-none text-gray-400"
+              aria-hidden="true"
+            />
+            {repo.stats.stars}
+          </p>
+        </div>
+        <ChevronRightIcon
+          className="h-5 w-5 flex-none text-gray-400"
+          aria-hidden="true"
+        />
+      </div>
+    </div>
+  );
+
+  const manageDelete = (repo) => (
+    <Delete action={handleDelete} id={repo._id}>
```

Comment:
```suggestion
    <Delete action={handleDelete} id={repo._id} label={repo.name}>
```

Also, It'd be great to pass in the `repo.name` to the `Delete` component `label` prop to complete the narrative the accessibility as `aria-label` attribute intends and will be used in the suggested `title` attribute too.

```js
aria-label={`Delete ${label}`}
title={`Delete ${label}`}
```

Reply:
Great idea, thank you 👍